index.bs

```diff
@@ -269,7 +289,17 @@ of an {{IDBTransaction}} |transaction|. To <dfn>calculate the effective <a spec=
 
 <h3 id="storage-bucket-caches">Using CacheStorage</h3>
 
-A {{StorageBucket}} has a {{CacheStorage}} object. [[service-workers#cache-objects]] objects created through {{StorageBucket/caches}} are associated with the bucket.
+A {{StorageBucket}} has a {{CacheStorage}} object. {{StorageBucket/caches}} getter steps are:
+
+1. Let |bucket| be [=this=]'s [=/storage bucket=].
+
+1. Let |bottle map| be the result of [=obtain a local storage bottle map=] with |bucket| and `"cacheStorage"`.
+
+1. Let |cacheStorage| be a {{CacheStorage}} object.
+
+1. Set the <a spec="service-workers">relevant name to cache map</a> for |cacheStorage| to |bottle map|.
```

Comment:
I wonder if this should be ` [=obtain a local storage bottle map=]` similar to indexedDB? `relevant name to cache map` looks like it might be keyed by `cacheName` within a bucket, and not a key for buckets.

Reply:
Hmmm, I believe there is some confusion here because

- a bucket has a "bottle map", i.e. map of bottle identifiers to bottles, mapping "cacheStorage" to a storage bottle for CacheStorage
- a storage bottle has a map, used for:

> A storage bottle’s map is where the actual data meant to be stored lives

The algorithm "obtain a local storage bottle map" refers to the (storage bottle)'s (map), rather than a (storage bucket)'s (bottle map).

Does this clear things up?

Reply:
Oh yes, somehow missed L302 😵 lgtm!